### ISSUE TITLE
How to convert bpf bytecode object into struct ebpf_inst

### ISSUE THREAD

**Issue by Dantali0n** [YutaroHayakawa/generic-ebpf#6]
Hello, 

I was wondering if there are any methods to convert bpf bytecode objects into `ebpf_inst`? Doing this manually would appear to be a rather painful process.

**Comment by YutaroHayakawa** [comment]
Yes, actually in Linux eBPF, generally it is performed by the component called loader. But currently, we don't have complete loaders, so we need to somehow extract the bytecode from ELF file by ourselves...
